### DISCUSSION TITLE
Start a reactive consumer paused

### DISCUSSION THREAD

**Discussion by danielcmas** [quarkusio/quarkus#31575] [Q&A]
Hello,
we have a requirement in which we need our Kafka consumers to start paused when the application is deployed, so we can control later if we want a consumer to resume/pause depending on certain conditions. 

Is is possible to start a reactive consumer in paused  status? I went through the docs but I haven't seen this possibility. I tried configuring the channel without any topic subscription, and then subscribe to the topic when we need to resume, but it seems that the channel always needs a topic when it's initialized, otherwise it will use a topic with the channel name.

I also saw that the channel can be disabled on configuration, but it seems it's not possible to enable it at runtime.

Thanks for the help

**Comment by scspijker** [comment]
We also have a use case for this feature, we would like to start paused and only start the consumer on custom signals.
